staging/src/k8s.io/client-go/metadata/metadata.go

```diff
@@ -64,7 +64,9 @@ func ConfigFor(inConfig *rest.Config) *rest.Config {
 	config := rest.CopyConfig(inConfig)
 	config.AcceptContentTypes = "application/vnd.kubernetes.protobuf,application/json"
 	config.ContentType = "application/vnd.kubernetes.protobuf"
-	config.NegotiatedSerializer = metainternalversionscheme.Codecs.WithoutConversion()
```

Comment:
I'm +1 on this change, but I think it needs to be done in several places, including:

* setDiscoveryDefaults
* dynamic/simple.go#ConfigFor
* the generated client setConfigDefaults methods

Reply:
All done! Thank you